src/GridList/GridTile.js

```diff
@@ -171,6 +176,7 @@ class GridTile extends Component {
       subtitle,
       titlePosition, // eslint-disable-line no-unused-vars
       titleBackground, // eslint-disable-line no-unused-vars
+      titleStyle, // eslint-disable-line no-unused-vars
```

Comment:
The property is used. You should remove `// eslint-disable-line no-unused-vars`.